### DISCUSSION TITLE
Tarteaucitron et Atinternet (mode hybride)

### DISCUSSION THREAD

**Discussion by ftrouve** [AmauriC/tarteaucitron.js#724] [General]
Bonjour,

j'utilise déjà sur plusieurs sites votre solution avec ATinternet cela fonctionne très bien.

Sur un nouveau site j'essaie de mettre la gestion privacy d'AT. Mode exempt sans consentement et refus cookies et opt'in si acceptation. Du coup, j'utilise push('atinternet') au lieu de 'atinternethightrack'

mais je n'arrive pas à ajouter le plan de marquage au tag déclaré dans tarteaucitron.services.js
Si je déclare un nouveau tracker.tag dans la function atMore, forcément j'ai 2 hits envoyés chez AT.
Comment faire donc pour ajouter dans la fonction atMore par ex un marquage très simple comme  : 
tag.page.set({
     name: document.title
 });

tarteaucitron.user.atMore = function () { 
     /* add here your optionnal ATInternet.Tracker.Tag configuration */
 };

(tarteaucitron.job = tarteaucitron.job || []).push('atinternet');

Merci d'avance 
Cordialement

**Comment by AmauriC** [comment]
Hey : )

Ajoutez l'option `<script>tarteaucitron.user.atinternetSendData = false;</script>` pour que tarteaucitron n'envoi plus de hit, il ne restera que le votre ; )

**Comment by ftrouve** [comment]
merci pour la réponse rapide.
Bonne journée
